src/app/dashboard/dashboard-sensor-details/dashboard-sensor-details.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+
+import CHART_PERIOD from '../chart-widget/chart-period.const';
+const template = require('./dashboard-sensor-details.template.html');
+const style = require('../dashboard.style.scss');
+
+const selector = 'dashboard-sensor-details';
+
+@Component({
+    selector,
+    template,
+    styles: [style]
+})
+export class DashboardSensorDetailsComponent {
```

Comment:
It will be better to rename this into something what describe main work of this component, like ```DeviceStatisticComponent```